spec/integration/commands/create_spec.rb

```diff
@@ -278,4 +280,64 @@ def self.[](input)
       }.to raise_error(ArgumentError, /user/)
     end
   end
+
+  describe '#execute' do
+    context 'with postgres adapter' do
+      context 'with a single record' do
+        subject! { container.command(:users).create.execute(name: 'Jane') }
+
+        it 'materializes the result' do
+          is_expected.to eq([
+            { id: 1, name: 'Jane' }
+          ])
+        end
+      end
+
+      context 'with multiple records' do
+        subject! do
+          container.command(:users).create.execute([
+            { name: 'Jane' },
+            { name: 'John' }
+          ])
+        end
```

Comment:
could you get rid of that and simply assign result to a local var in the example block?